tests/test_base_device.py

```diff
@@ -1176,6 +1055,7 @@ def test_process_estimator_job(self, measurements, expectation):
 
     @pytest.mark.parametrize("num_wires", [1, 3, 5])
     @pytest.mark.parametrize("num_shots", [50, 100])
+    @pytest.mark.skip(reason="Need to replace this with using SamplerV2.")
```

Comment:
Works as is without skipping but it calls backend.run which we don't really care about anymore.